components/eamxx/cime_config/testdefs/testmods_dirs/scream/scream_example_testmod_atmchange/shell_commands

```diff
@@ -1 +1 @@
-./xmlchange SCREAM_ATMCHANGE_BUFFER='cubed_sphere_map=42'
+./xmlchange --append SCREAM_ATMCHANGE_BUFFER='cubed_sphere_map=42'
```

Comment:
Didn't you drop support for `--append` in favor of `+=`?

Reply:
That `--append` is going to xmlchange, not atmchange.